src/frameworks/us_privacy_string_generator.js

```diff
@@ -0,0 +1,22 @@
+const FrameworkBase = require('./base');
+
+/**
+ * Implements usprivacy string framework
+ * for more information on the US Privacy string see:
+ * https://github.com/InteractiveAdvertisingBureau/USPrivacy/blob/master/CCPA/US%20Privacy%20String.md#us-privacy-string-format
+ */
+class UsPrivacyStringGenerator extends FrameworkBase {
+  supportedGenerators() {
+    return ['usPrivacyString'];
+  }
+
+  usPrivacyString(callback = () => {}, privacyComplianceInstance) {
+    let usp = '1';
+    usp += this.privacyComplianceInstance.hasBeenNotifiedOfRights() ? 'Y' : 'N';
+    usp += this.privacyComplianceInstance.canUsePersonalInformationForTargeting() ? 'N' : 'Y';
+    usp += this.privacyComplianceInstance.isLSPACoveredTransaction() ? 'Y' : 'N';
+    callback(usp);
```

Comment:
As per the PR description, should this not also be passing in `this` to the `callback()` call? Otherwise how will the caller distinguish between possible values?

Reply:
Oh great catch, this is what I get for letting modified code sit un-pushed locally. This has been updated.